core/internal/runconsolelogs/runconsolelogs.go

```diff
@@ -46,10 +45,17 @@ type Sender struct {
 }
 
 type Params struct {
+	// ConsoleOutputFile is the run file path to which to write captured
+	// console messages.
 	ConsoleOutputFile paths.RelativePath
 
-	Settings *settings.Settings
-	Logger   *observability.CoreLogger
+	// FilesDir is the directory in which to write the console output file.
+	FilesDir string
```

Comment:
This description isn't complete: it's not just the place to put the file (in which case it would be OK to set it to a temporary directory), but rather it's the root directory for all run files.

Reply:
yeah, correct, tried to separate the concepts, but maybe being more concrete is the right thing here, will update